src/components/tile-select/tile-select.component.js

```diff
@@ -74,15 +88,17 @@ const TileSelect = (props) => {
         </StyledTitleContainer>
         <StyledDescription>{description}</StyledDescription>
       </StyledTileSelect>
-      {checked && (
-        <StyledDeselectButton
-          buttonType="tertiary"
-          size="small"
-          onClick={handleDeselect}
-        >
-          {I18n.t("tileSelect.deselect", { defaultValue: "Deselect" })}
-        </StyledDeselectButton>
-      )}
+      <StyledDeselectWrapper
+        {...(actionButtonAdornment && {
+          mr: 2,
+          display: "flex",
+          alignItems: "center",
+          minHeight: "32px",
+        })}
```

Comment:
why is that css here? Couldn't it be moved to the styled component?

Reply:
The ticket said to use Box which has an interface that supports the styling already so I added it here instead of css